firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionToggleView.swift

```diff
@@ -0,0 +1,118 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+
+final class TrackingProtectionToggleView: UIView, ThemeApplicable {
+    private struct UX {
+        static let toggleLabelsContainerConstraintConstant = 16.0
+    }
+
+    var toggleSwitchedCallback: (() -> Void)?
+
+    private let toggleLabelsContainer: UIStackView = .build { stack in
+        stack.backgroundColor = .clear
+        stack.distribution = .fill
+        stack.alignment = .leading
+        stack.axis = .vertical
+        stack.spacing = TPMenuUX.UX.headerLabelDistance
```

Comment:
`TPMenuUX`

Reply:
This is used in two files only... I can duplicate it but for sure will not be the best option, from my perspective.